hack/autodoc/autodoc.go

```diff
@@ -0,0 +1,347 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const (
+	spaceMargin             = 10
```

Comment:
Consider externalizing configuration (to allow adjusting without recompiling etc).

Reply:
After discussion we decided to postpone this task to next MVP.